tests/useful_mocks.py

```diff
@@ -1,31 +1,55 @@
+from typing import Any
 from unittest import mock
 
-from screenpy.actions import Debug
```

Comment:
Even though i did like the cheekiness of importing `Debug` to be our dummy Action, i'm glad we're not doing it anymore. Nice work!